Source/WebCore/platform/network/soup/NetworkStorageSessionSoup.cpp

```diff
@@ -410,6 +410,26 @@ void NetworkStorageSession::setCookie(const Cookie& cookie)
     soup_cookie_jar_add_cookie(cookieStorage(), cookie.toSoupCookie());
 }
 
+void NetworkStorageSession::setCookieJar(const Vector<Cookie>& cookies)
```

Comment:
This is confusing, we have a method called `setCookieStorage` that receives a jar and this is called `setCookieJar` and receives a list of cookies. I would call this `setCookies` or `replaceCookies`